datasets/hst.yaml

```diff
@@ -0,0 +1,26 @@
+Name: HST public data
+Description: |
+  The Hubble Space Telescope (HST) is one of the most productive scientific instruments ever created. This dataset contains calibrated and raw data for all of the currently active instruments on HST: ACS, COS, STIS and WFC3. 
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Hourly
+Tags:
```

Comment:
@jedsundwall is there any guidance on what tags we should use here?

Reply:
Hi @arfon, you can see blessed tags at https://github.com/awslabs/open-data-registry/blob/master/tags.yaml. If you really think one should be added that's not in there, let us know. We're trying to strike a balance between being helpful but not overly descriptive.

Reply:
@jflasher Could "astronomy" be added to the tags? Biology, music, genomics, etc. are in the tags. This does not fit in "earth observations".

Reply:
Yep, I think that's fine. If you just want to add the tag to tags.yaml and include it in this PR, that'd work.

Reply:
👍done in https://github.com/spacetelescope/open-data-registry/commit/af7dff3a59756565f6e24a058bc7077707693d14